contracts/exchangeIssuance/ExchangeIssuance.sol

```diff
@@ -419,12 +419,11 @@ contract ExchangeIssuance is ReentrancyGuard {
         for (uint256 i = 0; i < components.length; i++) {
             uint256 scaledAmountEth = amountEthIn[i].mul(amountEth).div(sumEth);
             
-            uint256 amountTokenOut;
+            uint256 amountTokenOut = scaledAmountEth;
```

Comment:
Add a comment here making it obvious for the reader than `amountTokenOut = scaledAmountEth` when we are not performing a swap, and `exchange[i] == Exchange.Weth`.